master/buildbot/worker/latent.py

```diff
@@ -249,7 +250,11 @@ def insubstantiate(self, fast=False):
         try:
             yield d
         except Exception as e:
-            log.err(e, "while insubstanciating")
+            # The case of failure for insubstanciation is bad as we have a left-over costing ressource
+            # There is not much thing to do here generically, so we must put the problem of stop_instance
+            # reliability to the backend driver
+            log.err(e, "while insubstantiating")
```

Comment:
Yes, it does make sense, assuming the process for creating the thing is named "substantiate". It's also an awkward word and sort of requires the destroy process to use a word that doesn't exist.

I would not mind a renaming. OpenStack uses boot and delete, Docker has create/run and kill, while libvirt has create/define and delete. I like create and delete.